analytics/index.ts

```diff
@@ -0,0 +1,62 @@
+import { SDAI } from '@/constants';
+
+const formatEventName = (eventName: string) =>
+  eventName.split(' ').join('-').toLocaleLowerCase();
+
+export const FA_EVENTS = {
+  BETS: {
+    MY_BETS: 'click/bets/my-bets',
+    REDEEM: 'click/bets/redeem',
+  },
+  DEVCONFLICT: 'click/devconflict-banner',
+  FOOTER: {
+    CONNECT: {
+      DISCORD: 'click/footer/connect/discord',
+      GITHUB: 'click/footer/connect/github',
+      X: 'click/footer/connect/x',
+    },
+    HOME: 'click/footer/home',
+    MARKET_CATEGORY: (category: string) => `click/footer/markets/${category}`,
+    POWERED_BY: {
+      GNOSIS: 'click/footer/powered-by/gnosis',
+      GNOSIS_AI: 'click/footer/powered-by/gnosis-ai',
+      KLEROS: 'click/footer/powered-by/kleros',
+      REALITY: 'click/footer/powered-by/reality',
+    },
+  },
+  LIFI_WIDGET: {
+    OPEN: `click/lifi-widget/open-${SDAI.symbol}`,
+    ROUTE_FAILED: 'click/lifi-widget/route-failed',
+    ROUTE_SUCCESS: 'click/lifi-widget/route-success',
+  },
+  MARKETS: {
+    CATEGORY: (category: string) => `click/markets/category-${category}`,
+    DETAILS: {
+      EMBED: {
+        COPIED: 'click/markets/details/embed/copied',
+        ID: 'click/markets/details/embed',
+      },
+      ID: (id: string) => `click/markets/details/${id}`,
+      INFO: {
+        CONTRACT: 'click/markets/details/info/chain-scanner',
+        DISPUTE: 'click/markets/details/info/kleros',
+        ORACLE_ANSWER: 'click/markets/details/info/reality',
+      },
+      TABS: {
+        NAME: (tabName: string) => `click/markets/details/tabs/${tabName}`,
+        NEWS_ARTICLE: 'click/markets/details/tabs/news-article',
+      },
+    },
+    FILTERS: {
+      CREATOR: (creatorName: string) =>
+        `click/markets/filters/creator-${formatEventName(creatorName)}`,
+      ORDER: (orderName: string) =>
+        `click/markets/filters/order-${formatEventName(orderName)}`,
+      STATE: (stateName: string) =>
+        `click/markets/filters/state-${formatEventName(stateName)}`,
+      TOKEN: (tokenName: string) =>
+        `click/markets/filters/token-${formatEventName(tokenName)}`,
+    },
```

Comment:
Maybe having a way to know what filters are applied on the url is better than tracking every click? Also, should we track every option click or just clicking on each filter select?

Reply:
I didn't find a way to track search params from the URL yet. That would solve the market details tracking (as we could use the `?id` from the URL)

If we want to know if the users are using filters, we can drop the option clicks.
If we want to know the most popular filters, the current approach works.

Follows a similar heuristic to the one used for the market categories: registering a click on the categories control vs registering every category click.

Reply:
We can go forward with this solution